lib/services/large_result_set.js

```diff
@@ -64,11 +64,14 @@ function LargeResultSetService(connectionConfig, httpClient) {
     const callback = function callback(err, response, body) {
       // err happens on timeouts and response is passed when server responded
       if (err || isUnsuccessfulResponse(response)) {
-        // if we're running in DEBUG loglevel, probably we want to see the full error too
-        const logErr = err ? JSON.stringify(err, Util.getCircularReplacer())
-          : `status: ${JSON.stringify(response.status)} ${JSON.stringify(response.statusText)}` 
-            + ` headers: ${JSON.stringify(response.headers)}`;
-        Logger.getInstance().debug('Encountered an error when getting data from cloud storage: ' + logErr);
+        Logger.getInstance().debug('Encountered an error when getting data from cloud storage. Details: ');
+        if (err) {
+          // if we're running in DEBUG loglevel, probably we want to see the full error too
+          Logger.getInstance().debug(err);
```

Comment:
I'm not sure I like it. It will be logged in separate lines, right? It may be problematic for all tools for logs gathering (like ELK). Can we join a string and prepare a single message before logging?

Reply:
That's what we were doing up until now, and resulted in `{}` as the error, hence this PR :(

Reply:
But can't we do something like this?
```
errMsg = `Encountered blabla...`
if (err) {
   errMsg += err
} else {
   errMsg += `status: ...
}
Logger.getInstance().debug(err)
```

Reply:
resolved